Milad.css

```diff
@@ -0,0 +1,127 @@
+/* Reset some default styles */
+body, h1, h2, h3, p, ul, li {
+    margin: 0;
+    padding: 0;
+}
```

Comment:
I would switch line line 2 -5 to 7-15 organizing your CSS based on the hierarchy of your HTML structure is a recommended practice.

Like 
```
body { 
... code ...  
}

body, h1, h2, h3, p, ul, li {
    margin: 0;
    padding: 0;
} 
```